components/mediation-message-processor/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.wso2.carbon.</groupId>
+	<artifactId>mediation-message-processor</artifactId>
+	<packaging>bundle</packaging>
+	<version>4.4.1-SNAPSHOT</version>
+	<name>mediation-message-processor</name>
+	<url>http://maven.apache.org</url>
+	<dependencies>
+		<dependency>
+			<groupId>org.wso2.carbon.mediation</groupId>
+			<artifactId>org.wso2.carbon.message.processor</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.synapse</groupId>
+			<artifactId>synapse-core</artifactId>
+		</dependency>
+	</dependencies>
+	<parent>
+		<groupId>org.wso2.carbon.mediation</groupId>
+		<artifactId>carbon-mediation</artifactId>
+		<version>4.4.1-SNAPSHOT</version>
+		<relativePath>../../pom.xml</relativePath>
+	</parent>
+
```

Comment:
indentation issues were solved in pull request https://github.com/wso2/carbon-mediation/pull/258